bundles/org.openhab.binding.bluetooth.dbusbluez/src/main/java/org/openhab/binding/bluetooth/dbusbluez/handler/DBusBlueZBridgeHandler.java

```diff
@@ -0,0 +1,343 @@
+import org.openhab.binding.bluetooth.BluetoothAddress;
+import org.openhab.binding.bluetooth.dbusbluez.DBusBlueZBluetoothDevice;
+import org.openhab.binding.bluetooth.dbusbluez.internal.DBusBlueZUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.hypfvieh.bluetooth.DeviceManager;
+
+/**
+ * The {@link DBusBlueZBridgeHandler} is responsible for talking to the BlueZ stack, using DBus Unix Socket.
+ * This Binding does not use any JNI.
+ * It provides a private interface for {@link DBusBlueZBluetoothDevice}s to access the stack and provides top
+ * level adaptor functionality for scanning and arbitration.
+ *
+ * @author Benjamin Lafois - Initial contribution and API
+ */
+@NonNullByDefault
+public class DBusBlueZBridgeHandler extends AbstractBluetoothBridgeHandler<DBusBlueZBluetoothDevice> {
+
+    private final Logger logger = LoggerFactory.getLogger(DBusBlueZBridgeHandler.class);
+
+    // ADAPTER from BlueZ-DBus Library
+    private @NonNullByDefault({}) com.github.hypfvieh.bluetooth.wrapper.BluetoothAdapter adapter;
+    private @NonNullByDefault({}) com.github.hypfvieh.bluetooth.DeviceManager deviceManager;
+
+    // Our BT address
+    private @NonNullByDefault({}) BluetoothAddress adapterAddress;
+
+    private DBusBlueZPropertiesChangedHandler propertiesChangedHandler = new DBusBlueZPropertiesChangedHandler();
+
+    private final ReentrantLock lockDiscoveryJob = new ReentrantLock();
+
+    private @NonNullByDefault({}) ScheduledFuture<?> discoveryJob;
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+
+    /**
+     * Constructor
+     *
+     * @param bridge the bridge definition for this handler
+     */
+    public DBusBlueZBridgeHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        logger.debug(
+                "  _____  ____              ____  _            ______  ____  _            _              _   _      ");
+        logger.debug(
+                " |  __ \\|  _ \\            |  _ \\| |          |___  / |  _ \\| |          | |            | | | |     ");
+        logger.debug(
+                " | |  | | |_) |_   _ ___  | |_) | |_   _  ___   / /  | |_) | |_   _  ___| |_ ___   ___ | |_| |__   ");
+        logger.debug(
+                " | |  | |  _ <| | | / __| |  _ <| | | | |/ _ \\ / /   |  _ <| | | | |/ _ \\ __/ _ \\ / _ \\| __| '_ \\  ");
+        logger.debug(
+                " | |__| | |_) | |_| \\__ \\ | |_) | | |_| |  __// /__  | |_) | | |_| |  __/ || (_) | (_) | |_| | | | ");
+        logger.debug(
+                " |_____/|____/ \\__,_|___/ |____/|_|\\__,_|\\___/_____| |____/|_|\\__,_|\\___|\\__\\___/ \\___/ \\__|_| |_| ");
+        logger.debug("");
+
+        logger.debug("Initializing DBusBlueZBridgeHandler...");
```

Comment:
Bindings shouldn't log banners, especially when they could be logged more than once after startup.